demisto_sdk/commands/format/update_pre_process_rules.py

```diff
@@ -0,0 +1,63 @@
+import traceback
+from typing import Tuple
+
+import click
+
+from demisto_sdk.commands.common.hook_validations.pre_process_rule import \
+    PreProcessRuleValidator
+from demisto_sdk.commands.format.format_constants import (ERROR_RETURN_CODE,
+                                                          SKIP_RETURN_CODE,
+                                                          SUCCESS_RETURN_CODE)
+from demisto_sdk.commands.format.update_generic_json import BaseUpdateJSON
+
+PRE_PROCESS_RULES_PREFIX = 'preprocessrule-'
```

Comment:
not being used here, should be moved to constant.py (or removed)

Reply:
https://github.com/demisto/demisto-sdk/commit/b09b4300492be4222252252c9df22886d58e6a97